android/build.gradle

```diff
@@ -135,16 +135,29 @@ subprojects { project ->
       // TODO: (barthap) Replace this with raw string when dropped shell app macros
       // The star "*" signs interferes with slash "/" and treated wildcard path as comment ¯\_(ツ)_/¯
       targetExclude(["**", "versioned/host/exp/exponent/modules/api", "**", "*.kt"].join("/"))
-      ktlint("0.41.0").userData([
-        "disabled_rules"           : "no-wildcard-imports,import-ordering",
-        "charset"                  : "utf-8",
-        "end_of_line"              : "lf",
-        "indent_size"              : "2",
-        "indent_style"             : "space",
-        "insert_final_newline"     : "true",
-        "tab_width"                : "2",
-        "trim_trailing_whitespace" : "true"
-      ])
+      ktlint("1.0.1")
+        .editorConfigOverride([
+          "ktlint_standard_no-wildcard-imports"           : "disabled",
+          "ktlint_standard_import-ordering"               : "disabled",
+          "ktlint_standard_filename"                      : "disabled",
+          "ktlint_standard_property-naming"               : "disabled",
+          "ktlint_standard_discouraged-comment-location"  : "disabled",
+          "ktlint_standard_comment-wrapping"              : "disabled",
+          "ktlint_standard_function-naming"               : "disabled",
+          "ktlint_standard_class-naming"                  : "disabled",
+          "ktlint_standard_package-name"                  : "disabled",
+          "ktlint_standard_multiline-expression-wrapping" : "disabled",
+          "charset"                  : "utf-8",
+          "end_of_line"              : "lf",
+          "indent_size"              : "2",
+          "continuation_indent_size" : "2",
+          "indent_style"             : "space",
+          "insert_final_newline"     : "true",
+          "tab_width"                : "2",
+          "trim_trailing_whitespace" : "true",
+          "ij_kotlin_allow_trailing_comma_on_call_site": "false",
+          "ij_kotlin_allow_trailing_comma": "false"
```

Comment:
is that intentional to remove the trailing comma? it looks like that break previous style, e.g. see the change in **ExpoUpdatesAppLoader.kt**.

Reply:
With that set to `true`, `ktlint` will try to use trailing commas everywhere. It's better to turn that off for now; otherwise, it will affect most of our codebase.

Reply:
oh makes sense. thanks for pointing out this